dts/bindings/rtc/nxp,irtc.yaml

```diff
@@ -40,3 +39,9 @@ properties:
       3 <- Coarse 1Hz
       Default value ensures the output clock is turned off.
       This is the reset value.
+  
+  share-counter:
+    type: boolean
+    description: |
+      This slave irtc instance shares the data and time counters of the master irtc instance. 
```

Comment:
use a different term than "slave" and "master". See https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html#rule-a-2-inclusive-language